### PR TITLE
Change instructions to install from source

### PR DESCRIPTION
**Pull request by diasjorge**

### CODE REVIEW

File: installation/from_source_repository.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+If you want to contribute then you might want to install Crystal from sources.
 
-You will also need LLVM 3.5 or 3.6 present in the path. If you are using Mac and the Homebrew formula, this will be automatically configured for you if you install Crystal adding `--with-llvm` flag.
+1. [Install the latest Crystal release](https://crystal-lang.org/docs/installation). To compile Crystal, you need Crystal :).
 
+2. Make sure LLVM 3.5 or 3.6 is present in the path. If you are using Mac and the Homebrew formula, this will be automatically configured for you if you install Crystal adding `--with-llvm` flag.
 
-1. Make sure to install [all the required libraries](https://github.com/crystal-lang/crystal/wiki/All-required-libraries). You might also want to read the [contributing guide](https://github.com/crystal-lang/crystal/blob/master/CONTRIBUTING.md).
+3. Make sure to install [all the required libraries](https://github.com/crystal-lang/crystal/wiki/All-required-libraries). You might also want to read the [contributing guide](https://github.com/crystal-lang/crystal/blob/master/CONTRIBUTING.md). You will also need to have a released version of Crystal already installed.
```
Comment (mverzilli) [inline installation/from_source_repository.md]: Let's remove this last sentence, I think that's already clear as it is listed in the first step.

File: installation/from_source_repository.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
 # From sources
 
-If you want to contribute then you might want to install Crystal from sources. But Crystal is written in Crystal itself! So you first need to use one of the previous described methods to have a running compiler.
+If you want to contribute then you might want to install Crystal from sources.
 
-You will also need LLVM 3.5 or 3.6 present in the path. If you are using Mac and the Homebrew formula, this will be automatically configured for you if you install Crystal adding `--with-llvm` flag.
+1. [Install the latest Crystal release](https://crystal-lang.org/docs/installation). To compile Crystal, you need Crystal :).
 
+2. Make sure LLVM 3.5 or 3.6 is present in the path. If you are using Mac and the Homebrew formula, this will be automatically configured for you if you install Crystal adding `--with-llvm` flag.
```
Comment (mverzilli) [inline installation/from_source_repository.md]: This is a bit out of date. We already support LLVM 3.9 and we should encourage new users to use it if possible. So I'd suggest:

"Make sure a supported LLVM version is present in the path. Currently, Crystal supports LLVM 3.5, 3.6, 3.8 and 3.9. When possible, use the latest one. If you are using Mac [...]"

**Comment by diasjorge** [comment]
@mverzilli Is this good to merge now?

**Comment by mverzilli** [comment]
Hi @diasjorge, sorry for the delay, took vacations 😄 . We're deprecating LLVM < 3.8 and supporting LLVM 4.0, so could you update the PR to specify that versions 3.8, 3.9 and 4.0 of LLVM are required?

**Comment by diasjorge** [comment]
@mverzilli I hope you enjoyed those holidays :)
I've updated the requirements, let me know if there's anything else required.

**Comment by mverzilli** [comment]
Thanks for the contribution and your patience ❤️

**Pull request merged**